### PR TITLE
add feature switch for HomeView

### PR DESCRIPTION
**Pull request by Roshanjossey**
<h2>PR checklist</h2>

**Docs**
- [x] If you added new code; is it self documenting? (all the variable names clear, comments added where appropriate)
- [x] If you touched existing code; did you refactor existing naming, added comments where necessary?

**Tests**
- [x] If your ticket adds new functionality; have you added unit tests to verify behaviour?
- [ ] If your ticket alters user interaction with the UI; have you added integration tests?
- [x] Do your tests check for every _convievable_ user behabviour? (all behaviours the user _can_ perform)

**Typing**
- [x] If you added new code; have you added typing?
- [ ] If you touched existing code; have you improved its typing?

**Styling**
- [ ] If new styles were created; Did you refactor/add more styles in css-in-js style?

### CODE REVIEW

File: src/helpers/featureSwitch.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+type StringMap = { [key: string]: any };
+
+export const getFeatureFlags = (environmentVariables: StringMap) => {
+    return Object.fromEntries(
+        Object.entries(environmentVariables)
+        .filter(variableEntry => variableEntry[0].startsWith("REACT_APP_FEATURE"))
```
Comment (BastinJafari) [inline src/helpers/featureSwitch.ts:6]: [envVarName, envVarValue] => envVarName.startsWith("REACT_APP_FEATURE"))

**Review by BastinJafari — CHANGES_REQUESTED** [review]
Awesome PR. Just one small thing

File: src/helpers/featureSwitch.ts (hunk 1 above)
Comment (BastinJafari) [reply src/helpers/featureSwitch.ts:6]: Or something similar to make it more readable?

Comment (Roshanjossey) [reply src/helpers/featureSwitch.ts]: @BastinJafari, updated

**Review by BastinJafari — APPROVED** [review]

**Pull request merged**